Lib/Src/CParser.cpp

```diff
@@ -0,0 +1,309 @@
+/**
+ * @file CParser.cpp
+ */
+/*
+ * Created on : Apr 4, 2022
+ * Author : niuslar
+ */
+
+#include "CParser.h"
+
+#define ERROR_VALUE (0.00)
+
+/**
+ * @brief Constructor
+ */
+CParser::CParser() {}
+
+/**
+ * @brief Parse string
+ * @param String to be parsed
+ * @return Last parsing state
```

Comment:
This is very vague and requires more explanation. Why are you returning last parsing state? How is the user meant to utilise it?